lib/phoenix/test/channel_test.ex

```diff
@@ -239,9 +239,12 @@ defmodule Phoenix.ChannelTest do
   """
   defmacro connect(handler, params) do
     if endpoint = Module.get_attribute(__CALLER__.module, :endpoint) do
+      stringified_params =
+        Code.eval_quoted(params) |> elem(0) |> stringify_params |> Macro.escape
```

Comment:
This won't work if you have, for example, a variable in parameter. The proper solution would be to do something like:

    SomeModule.stringify_keys(unquote(params))

The question is where to define stringify_keys so we can access it for such but without importing it into the user scope